tests/unit/models/DefaultData.spec.js

```diff
@@ -1083,4 +1088,198 @@ describe('Test class: DefaultData', () => {
       expect(pluginData.getEventLogById(id3)).not.toBeUndefined();
     });
   });
+
+  describe('Test method: getAttributeValue', () => {
+    it('Should get the value of a variable attribute', () => {
+      const pluginData = new DefaultData();
+      const attribute = new ComponentAttribute({
+        name: 'test_attribute',
+        value: 'test_variable',
+      });
+
+      pluginData.variables.push(new Variable({
+        name: 'test_variable',
+        value: 'test',
+      }));
+
+      Object.defineProperty(attribute, 'isVariable', {
```

Comment:
why not `attribute.isVariable = true` ?

Reply:
Because otherwise it triggers this error: 
`TypeError: Cannot set property isVariable of #<ComponentAttribute> which has only a getter`